features/supplier/supplier_applies_to_a_framework.feature

```diff
@@ -34,3 +34,29 @@ Scenario: Supplier submits a framework application
   And I click 'Back to framework application' link for that framework application
 
   And I see 'Your application will be submitted at' text on the page
+
+Scenario: Supplier re-uses a declaration
+  Given I have a supplier with a reusable declaration
+  And that supplier has begun the application process for that framework
+  And that supplier has confirmed their company details for that application
+  And that supplier has a user with:
+    |active|true|
+  And that supplier is logged in
+  When I visit the /suppliers page
+  And I click 'Continue your application'
+  Then I am on the 'Apply to framework' page for that framework application
+
+  When I click 'Make supplier declaration'
+  Then I am on the 'Make your supplier declaration' page
+  When I click 'Start your declaration'
+  Then I am on the 'Reusing answers from an earlier declaration' page
+
+  When I choose the 'Yes' radio button for the 'Do you want to reuse the answers from your earlier declaration?' question
+  And I click 'Save and continue'
+  Then I am on the 'Your declaration overview' page
+
+  When I answer all questions on that page and those following until I'm back here
```

Comment:
For this scenario we care less about answering the remaining questions, and more that at least one answer from the previous declaration has been copied. The answering of questions takes ages and we have coverage for that in the other scenario.

If we save the old declaration into `@reused_declaration` or similar, then we can check which questions have been excluded from being copied to the new framework (... from the frameworks repo manifest? 🤔 Might have to hardcode these...), see what answers are left in `@reused_declaration`, check that those questions have the link 'Review answer', follow the links and check that the expected values are there.

Reply:
Yup - this is the meat of the issue :meat_on_bone: . Will think about this.

Reply:
> The answering of questions takes ages and we have coverage for that in the other scenario.

But wasn't the bug we encountered to do with ending up with an un-validatable form state when reusing a particular declaration? So my instinct was to check that it was possible to successfully submit a declaration in this manner.

Reply:
The validation happens after each page is submitted - the bug with the reuse was because the copied declaration was in an invalid state to start off with, so when the user tried to answer any question (apart from the one causing the problem), they got an error.